templates/tower-project.yaml

```diff
@@ -188,6 +184,29 @@ Resources:
       Roles:
         - !Ref TowerForgeBatchHeadJobRole
 
+  TowerForgeExternalS3AccessPolicy:
+    Type: AWS::IAM::Policy
+    Properties:
+      PolicyName: TowerForgeExternalS3AccessPolicy
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Sid: ConditionalAmazonS3ReadOnlyAccess
+            Effect: Allow
+            Action:
+              - "s3:Get*"
```

Comment:
Great addition - its too bad the `ManagedPolicyArns` doesn't take a mapping where you can specify the Condition.